app/js/view/tabs_view.js

```diff
@@ -15,14 +15,14 @@ export default class TabsView extends View {
   }
 
   onChange(evt) {
-    var selected = this.tabs.selected === 0 ? 'Apps' : 'Add-ons';
+    var selected = this.tabs.selected === 0 ? 'apps' : 'addons';
     this.changeHandlers.forEach(handler => {
       handler(selected);
     });
   }
 
-  render() {
-    super();
+  render(initialTab) {
+    super([initialTab === 'addons' ? 1 : 0]);
```

Comment:
The '1' and '0' values for selected index in gaia-tabs is a little magical. But I'm not sure how we can make this more semantic.